Makefile

```diff
@@ -322,7 +322,10 @@ build/chart/index.yaml.$(YEAR_MONTH_DAY): build/chart/index.yaml
 
 build/chart/: build/chart/index.yaml build/chart/index.yaml.$(YEAR_MONTH_DAY)
 
-install-large-chart: build/toolchain/bin/helm$(EXE_EXTENSION) install/helm/open-match/secrets/
+install-chart-prerequisite: build/toolchain/bin/kubectl$(EXE_EXTENSION)
+	$(KUBECTL) apply -f install/open-match-metadata-server.yaml
```

Comment:
Rename to `gke-metadata-server-workaround.yaml` so it's a bit clearer that this will eventually go away.